build.gradle

```diff
@@ -44,6 +46,11 @@ configurations {
 
 dependencies {
   ktlint project(':ktlint')
+  subprojects.forEach { subproject ->
+    if (subproject.name != 'ktlint-ruleset-template') {
```

Comment:
This is pretty dopey, but excluding the project in the `shadowJar` config as described in this issue https://github.com/johnrengelman/shadow/issues/39 doesn't seem to work anymore.

Reply:
But what's the reasoning behind including the template module into the project? I mean `settings.gradle`

Reply:
Presumably to ensure it keeps building, but maybe it just causes more issues than it solves.

Reply:
Could be a separate pipeline maybe